### DISCUSSION TITLE
How to Place Recently Accessed Data in L2 Cache in gem5?

### DISCUSSION THREAD

**Discussion by AlfiRam** [gem5/gem5#1811] [Q&A]
Is there a way in gem5 to simulate explicitly placing recently accessed data into the L2 cache? Can this be done using existing mechanisms or would it require custom modifications? Looking for suggestions or examples.

**Comment by giactra** [comment]
Hi @AlfiRam, I am afraid you would have to share more details.
What do you mean by "explicitly"? 
Do you consider the automatic line fill caused by a cache miss as explicit?
If yes, do you want data to be inserted in L2 or **exclusively** in L2 (and not L1 as an example)?
